bin/qc-headers

```diff
@@ -244,14 +243,60 @@ def get_header(filename):
     Ensures the input dicom exists, and then attempts to parse it.
     """
     if not os.path.isfile(filename):
-        sys.exit('ERROR: input {} does not exist'.format(filename))
+        logger.error('ERROR: input {} does not exist'.format(filename))
+        sys.exit(1)
     try:
         hdr = dcm.read_file(filename)
         return hdr
     except:
-        sys.exit('ERROR: failed to parse {}'.format(filename))
+        logger.error('ERROR: failed to parse {}'.format(filename))
+        sys.exit(1)
+
+def read_bvecs(filename):
+    try:
+        with open(filename, 'r') as f:
+            a = np.fromstring(''.join(f.read().splitlines()),sep=" ").flatten()
+            return a
+    except:
+        logger.error('ERROR: failed to parse {}'.format(filename))
+        sys.exit(1)
+
+
+def check_bvecs(cmpdcm, stddcm):
 
-def main(dicom, standard, output):
+    Val = collections.namedtuple('Val', ['source', 'path'])
+    compare={'bval':True, 'bvec':True}
+
+    #i can't decide if this is sketchy
+    vals= {'bval':[Val(source="gold standard", path=stddcm.replace('.dcm','.bval')),
+        Val(source="comparison",  path=cmpdcm.replace('.dcm', '.bval').replace('/dcm/' , '/nii/'))],
+        'bvec':[Val(source="gold standard",  path=stddcm.replace('.dcm','.bvec')),
+        Val(source="comparison", path=cmpdcm.replace('.dcm', '.bvec').replace('/dcm/' , '/nii/'))]}
+
+    mismatches=[]
+
+    for type in vals:  #for each of bvals and bvecs
+        for x in vals[type]: #for each path
+            if not os.path.isfile(x.path):
+                compare[type]=False  #don't compare the files bc one of them doesnt exist
+                logger.error('ERROR: {} {} file not found.'.format(x.source,type))
+                mismatches.append(Mismatch(header="{}".format(type), expected="expected", actual="{} file not found!!".format(x.source), tolerance="None"))
+        if compare[type]==True:
+                if not np.array_equal(read_bvecs(vals[type][0].path), read_bvecs(vals[type][1].path)): #there will only ever be two input files so i'm probably safe
+                    mismatches.append(Mismatch(header=type, expected="expected", actual="not matched!!", tolerance="None"))
```

Comment:
I'm not sure how valuable it would be but it might be a good idea to print the actual bvecs/bvals values as 'expected' and 'actual' when a difference is uncovered. It would be worth asking Erin if that's worth the effort